.travis.yml

```diff
@@ -162,9 +162,9 @@ matrix:
         apt:
             sources:
                 - ubuntu-toolchain-r-test
-                - llvm-toolchain-trusty-5.0
+                - llvm-toolchain-xenial-6.0
             packages:
-                - clang-5.0
+                - clang-6.0
```

Comment:
clang-7.0

Reply:
Actually, if we have `llvm-toolchain-xenial-7` we don't even need to specify it as a package here